app/uk/gov/hmrc/tai/connectors/JourneyCacheConnector.scala

```diff
@@ -36,6 +36,7 @@ class JourneyCacheConnector @Inject()(httpHandler: HttpHandler, servicesConfig:
   def currentCache(journeyName: String)(implicit hc: HeaderCarrier): Future[Map[String, String]] =
     httpHandler.getFromApiV2(cacheUrl(journeyName)).map(_.as[Map[String, String]]) recover {
       case e: HttpException if e.responseCode == NO_CONTENT => Map.empty[String, String]
+      case e: HttpException if e.responseCode == NO_CONTENT => Map.empty[String, String]
```

Comment:
this is duplicated, from above isn't it?

Reply:
Yes. I was going to change it  but then left it as decided to restrain to just Controllers. Will remove .